app/stacks/cumulus/resources/granules/css/nga/WV04/1B/2017/125/WV04_594e4035-a94b-4da5-a5a2-9593702d7739-inv_X1BS_059102583030_01/WV04_20170505053805_594e4035-a94b-4da5-a5a2-9593702d7739-inv_17MAY05053805-P1BS-059102583030_01_P001-cmr.json

```diff
@@ -0,0 +1,188 @@
+                    "Algorithm": "SHA-256"
+                },
+                "MimeType": "image/tiff"
+            },
+            {
+                "Name": "WV04_20170505053805_594e4035-a94b-4da5-a5a2-9593702d7739-inv_17MAY05053805-P1BS-059102583030_01_P001.xml",
+                "SizeInBytes": 1137927,
+                "FormatType": "Native",
+                "Format": "XML",
+                "Checksum": {
+                    "Value": "0c45e2288b76005f075cf6a09658084fefbb363028e3e9a670fc214b1d540906",
+                    "Algorithm": "SHA-256"
+                },
+                "MimeType": "application/xml"
+            }
+        ]
+    },
+    "TemporalExtent": {
+        "SingleDateTime": "2017-05-05T05:38:05.144570Z"
+    },
+    "SpatialExtent": {
+        "HorizontalSpatialDomain": {
+            "Geometry": {
+                "GPolygons": [
+                    {
+                        "Boundary": {
+                            "Points": [
+                                {
+                                    "Longitude": 73.83583656,
+                                    "Latitude": 30.24332069
+                                },
+                                {
+                                    "Longitude": 73.83637995,
+                                    "Latitude": 30.12225413
+                                },
+                                {
+                                    "Longitude": 74.00085472,
+                                    "Latitude": 30.10068596
+                                },
+                                {
+                                    "Longitude": 74.00108314,
+                                    "Latitude": 30.22127221
+                                },
+                                {
+                                    "Longitude": 73.83583656,
+                                    "Latitude": 30.24332069
+                                }
+                            ]
+                        }
+                    }
+                ]
+            }
+        }
+    },
+    "MeasuredParameters": [
+        {
+            "ParameterName": "Top of Atmosphere (TOA) visible reflectance"
+        }
+    ],
+    "Platforms": [
+        {
+            "ShortName": "WORLDVIEW-4",
+            "Instruments": [
+                {
+                    "ShortName": "WV110"
+                }
+            ]
+        }
+    ],
+    "Projects": [
+        {
+            "ShortName": "CSDA"
+        }
+    ],
+    "RelatedUrls": [
+        {
+            "URL": "https://csdap-maxar-delivery.s3.us-west-2.amazonaws.com/WV04_20170505053805_594e4035-a94b-4da5-a5a2-9593702d7739-inv_17MAY05053805-P1BS-059102583030_01_P001-BROWSE.jpg",
+            "Type": "GET DATA",
+            "Description": "Browse Imagery",
+            "Format": "JPEG",
+            "MimeType": "image/jpeg",
+            "Size": 142286.0,
+            "SizeUnit": "NA"
+        },
```

Comment:
We also need another entry for the `thumb` file, which should be nearly identical to this entry, so make a copy of this entry, and in the copy, change the URL to use `thumb` instead of `BROWSE`, and change GET DATA to GET RELATED VISUALIZATION.

Reply:
Just did this as per instructions.
Thanks